fieldservice_route/views/res_partner_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Activar rutas desde el contacto -->
+    <record id="fsm_partner_fields" model="ir.ui.view">
```

Comment:
```suggestion 
    <record id="fsm_partner_fields_inherit" model="ir.ui.view">
        <field name="name">res.partner.fsm.form.inherit</field>
```

Reply:
As a personal convention, I prefer to assign the same id to the inherited view, so when I search in the code or in xml_ids they show up immediately